tox.ini

```diff
@@ -10,13 +10,14 @@ deps =
 commands = python -m pytest -v {posargs} test/functional
 sitepackages = True
 
-[testenv:integration]
-deps =
-    pytest
-    pyyaml
-basepython = python3
-commands = python -m pytest -v {posargs} test/integration
-sitepackages = False
+;; disabled during transition to ansiblized Taskotron (Standard Interface support)
```

Comment:
What is the actual reason to disable the tests? If I change the integration tests to run the ansible playbook in mock, would that work?

Reply:
On it.

Reply:
OK, there is a way, I have a very rough code ready, testing it.

Reply:
I have no idea :-) The reason I disabled it is because I don't have time ATM to implement running the test suite in docker/mock/something else, and I hoped you'll either don't care that much (since with ansible playbook, taskotron runner is now a *very* thin wrapper around all of this), or implement it yourself. Or I might get to it, eventually. Just not immediately.

Reply:
pushed, let's see what Travis says